src/cloudai/workloads/ai_dynamo/ai_dynamo.sh

```
@@ -124,301 +98,375 @@ _resolve_host_ip() {
-_apply_sglang_section_args() {
-  prefill_args["--port"]=${dynamo_args["prefill-port"]}
-  decode_args["--port"]=${dynamo_args["decode-port"]}
-  prefill_args["--served-model-name"]=${dynamo_args["model"]}
-  decode_args["--served-model-name"]=${dynamo_args["model"]}
-
-  # model-path must point to HF cache for sglang
-  prefill_args["--model-path"]="${HUGGINGFACE_HOME}"
-  decode_args["--model-path"]="${HUGGINGFACE_HOME}"
-
+_apply_sglang_dsr1_section_args() {
   local self="$(_current_node_name)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)